lib/child_process.js

```diff
@@ -403,8 +403,9 @@ function normalizeSpawnArguments(file, args, options) {
 
   if (Array.isArray(args)) {
     args = args.slice(0);
-  } else if (args !== undefined &&
-             (args === null || typeof args !== 'object')) {
+  } else if (args === undefined) {
```

Comment:
Any reason not to be generous and permit `null` here? I believe ur linter permits `== null` which evaluates to `true` for both `null` and `undefined`. In fact, there are two other places in this file already that use that.

Reply:
Adjusted that case.